tests/unit/cn/mbset_test.c

```diff
@@ -132,8 +84,22 @@ post(struct mtf_test_info *mtf)
     return 0;
 }
 
+typedef merr_t
+mbset_udata_init_fn(const struct kvs_mblk_desc *mblk, void *rock);
+
+static merr_t
+t_udata_init(const struct kvs_mblk_desc *mblk, void *rock)
+{
+    struct udata *u = rock;
+
+    u->mbid = mblk->mbid;
+    return u->return_code;
+}
+
+
 MTF_BEGIN_UTEST_COLLECTION_PREPOST(test, init, fini);
 
+#if 0
```

Comment:
What is the status on this?

Reply:
oops.  i will investigate...

Reply:
fixed these unit tests.  good catch.  keeping me honest :)